panoramapublic/src/org/labkey/panoramapublic/PanoramaPublicModule.java

```diff
@@ -65,7 +65,7 @@ public String getName()
     @Override
     public @Nullable Double getSchemaVersion()
     {
-        return 19.31;
+        return 20.32;
```

Comment:
It's totally fine to call this 20.32, but we've recently switched from versioning the schema to the LabKey release to the year that the script was added.

https://www.labkey.org/Documentation/wiki-page.view?name=sqlScripts#ver

There's absolutely no need to change this here, but wanted to make sure you saw the newer approach.